src/main/java/com/fcgl/madrid/forum/model/Post.java

```diff
@@ -30,7 +31,8 @@
     public Post() {
```

Comment:
Think this should be removed as we should force the setting of the values when creating a Post. What do you think?

Reply:
I think this is required for Spring, can try without it and see if it works

Reply:
`An empty constructor is needed to create a new instance via reflection by your persistence framework.`

https://stackoverflow.com/questions/18099127/java-entity-why-do-i-need-an-empty-constructor

Reply:
Ok cool, but then shouldn't there be one for the comment class aswell?

Reply:
The comment class does have one

Reply:
oh sorry. completely missed it. will test it now